CHANGELOG.md

```diff
@@ -4,7 +4,7 @@ All notable changes to this project will be documented in this file.
 ## [unreleased]
```

Comment:
@meganemura This line should be also fixed because v1.4.1 was released

Reply:
@willnet It will be updated by #424.

Reply:
👍